### DISCUSSION TITLE
Can you share the SAM dataset with caption and JorneyDB dataset?

### DISCUSSION THREAD

**Discussion by Luo-Yihong** [PixArt-alpha/PixArt-alpha#91] [General]
Can you share the SAM dataset with caption and JorneyDB dataset?

**Comment by lawrence-cj** [comment]
We have released SAM-LLaVA here: https://huggingface.co/datasets/PixArt-alpha/SAM-LLaVA-Captions10M.

**Reply by Luo-Yihong** [reply]
Thanks for your reply! I am wondering do you re-caption the JourenyDB dataset? Or you just use the original caption provided by JourenyDB?

**Reply by ChongjianGE** [reply]
we use the original caption (short caption) for the 3rd training stage in Pixart-alpha.

**Reply by Luo-Yihong** [reply]
Is the key 'Prompt' or the key 'Caption' in the jsonl file of JorneyDB?
